multinetjs/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "multinet",
+  "version": "0.1.0",
+  "description": "Multinet client library",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "start": "tsc -w",
+    "build": "tsc",
+    "lint": "tslint -p tsconfig.json -c tslint.json",
+    "lint:fix": "tslint --force -p tsconfig.json -c tslint.json",
+    "test:server:up": "cd .. && PIPENV_DONT_LOAD_ENV=1 pipenv run test-server-up",
+    "test:server:down": "cd .. && PIPENV_DONT_LOAD_ENV=1 pipenv run test-server-down",
+    "test:server:clean": "cd .. && PIPENV_DONT_LOAD_ENV=1 pipenv run test-server-clean",
```

Comment:
It's needed on these commands because the .env file assumes you're working on the dev/production version of the server which gets served on a Flask-standard port (5000). I'm not totally sure which `PIPENV_DONT_LOAD_ENV`s can be omitted but it doesn't hurt to have them in place here.

I'm also not 100% that we need the one on the `test-server-up` script *and* in start.sh but again, the redundancy doesn't hurt. If you want to get rid of one or the other and see if the testing still works properly, that would be fine. Let me know what you find out!

Reply:
I just tested removing `PIPENV_DONT_LOAD_ENV` and it has no affect on the outcomes. The port is still set to 58529 for arango and to 50000 for the flask app. However, it does read the .env in but then doesn't use it when it launches the server. I think it's okay to leave it as it is for now

Reply:
You reminded me of why I did it this way: it's to stop the "loading .env" message when this command runs. I figured that message was misleading and should therefore be suppressed. I agree with leaving this as is. Thanks for looking into it!